queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered AND escape_attempts < 3;
```

Comment:
- [OPTIONAL]Kindly make sure you specify the `true` value of the neutered record, It might be like:

`SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < '3';`